.gitignore

```diff
@@ -1,5 +1,4 @@
 # Generated manuscript output files
-output/index.html
```

Comment:
It is only temporary.  I plan to remove it before merging.  I was tracking it so that I could use git diff to assess whether my local build in the version with templates changed the HTML content and show others that the author order, affiliations, etc. did not change during this pull request.